Structure_Engine/Query/Geometry.cs

```diff
@@ -101,8 +101,8 @@ public static CompositeGeometry Geometry(this ConcreteSection section)
                 return null;
 
             CompositeGeometry geom = Engine.Geometry.Create.CompositeGeometry(section.SectionProfile.Edges);
-            if(section.Reinforcement != null)
-                geom.Elements.AddRange(section.Layout().Elements);
+            //if(section.Reinforcement != null)
+                //geom.Elements.AddRange(section.Layout().Elements);
```

Comment:
Is this code relevant?

Reply:
good catch! Actually, could be, if we want the reinforcement to be automatically displayed. question is if we want to or not, and ofc, it varies across the length (or can vary). Torn if to add it in or not! :)